src/layout/Header.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from "react";
+import "./Header.css";
+
+import { Layout, Menu } from "antd";
+import { AuthContext } from "../Auth";
+import { LogoutOutlined, UserOutlined } from "@ant-design/icons";
+import SubMenu from "antd/lib/menu/SubMenu";
+import { Link } from "react-router-dom";
+
+const { Header: AntHeader } = Layout;
+
+function Header() {
+  const authCtx = useContext(AuthContext);
+
+  return (
+    <AntHeader className="header">
+      <Link className="header-logo" to="/">
+        <img
+          className="header-logo-image"
+          src="/logo_white.svg"
+          alt="Capact Logo"
+        />
+        <b className="header-logo-title">Capact</b>
+      </Link>
+      <Menu theme="dark" mode="horizontal" className="header-menu">
```

Comment:
the dark theme uses `#001529` for background, where for header use set `var(--capact-blue-dark);}` which is `#000c19`. We can see this diff on both edges:

![Screen Shot 2022-01-13 at 16 34 27](https://user-images.githubusercontent.com/17568639/149360148-4bcce569-dda5-4d77-a158-14578c4e0017.png)
![Screen Shot 2022-01-13 at 16 34 23](https://user-images.githubusercontent.com/17568639/149360295-ab7114ab-8427-4515-98c2-0bfde35d3f4e.png)


BTW, IMO `var(--capact-blue-dark);}`  is not too dark for our header

Reply:
![tenor-260367692](https://user-images.githubusercontent.com/7155799/149368522-f2ca603e-551b-4d89-a6b1-c5a977417258.gif)

Good catch - I didn't notice that, probably because of my display configuration... need to double check whether it is properly calibrated 😄